init.lua

```diff
@@ -204,30 +192,38 @@ for name, selection_box in pairs(selection_boxes) do
 				fixed = selection_box,
 			},
 			groups = {cracky = 3, oddly_breakable_by_hand = 3},
-			use_texture_alpha = true,
+			use_texture_alpha = "clip",
```

Comment:
"clip" makes stained glass opaque, "blend" should be used to have semi-transparent texture but then we have another strange bug under certain angles possibly due to nodebox thickness being 0.

Glass with `use_texture_alpha = "clip"`:
<img width="898" height="750" alt="screenshot_20250916_165918" src="https://github.com/user-attachments/assets/a48f2aae-68a3-4d1c-859c-394374bc098f" />

Glass with `use_texture_alpha = "blend"`: 
<img width="898" height="750" alt="screenshot_20250916_165935" src="https://github.com/user-attachments/assets/87e39183-2eaf-4b63-bbb1-b1846d1e1df4" />
You can see rightmost part is slightly lighter.